ipaserver/install/bindinstance.py

```diff
@@ -768,6 +768,13 @@ def __enable(self):
             logger.debug("Unable to mask named (%s)", e)
 
     def __setup_sub_dict(self):
+        if paths.NAMED_CRYPTO_POLICY_FILE is not None:
+            crypto_policy = 'include "{}";'.format(
```

Comment:
`named.conf` man pages don't describe the `include` statement 🤔

Reply:
IPA uses include already, see https://github.com/freeipa/freeipa/blob/release-4-6-3/install/share/bind.named.conf.template#L42-L45

It's documented here, http://www.zytrax.com/books/dns/ch7/include.html

Reply:
Thanks.